deleteCollectionMongo.js

```diff
@@ -21,3 +21,5 @@ async function deleteCollection() {
 }
 
 deleteCollection();
+
+//was trying to delete order collection but it threw an error Error deleting collection: TypeError: Order.deleteMany is not a function- didn't find the solution
```

Comment:
I think because the Order file exports an object rather than a single value. So you'll need to import it by doing `const { Order } = require('./src/models/Order');

see example in controller: https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/paymentEmail/src/controllers/orderController.js#L1
see export here : https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/paymentEmail/src/models/Order.js#L130

Reply:
@akosasante thank you so much for the explanation!!!